demos/python_demos/face_recognition_demo/face_recognition_demo.py

```diff
@@ -70,7 +70,14 @@ def build_argparser():
                         help="Path to the Facial Landmarks Regression model XML file")
     models.add_argument('-m_reid', metavar="PATH", default="", required=True,
                         help="Path to the Face Reidentification model XML file")
-
```

Comment:
I suggest returning that blank line back before `infer = parser.add_argument_group('Inference options')`.